hack/release/prepare-assets.sh

```diff
@@ -86,4 +86,10 @@ export IMG_NAME=projects.registry.vmware.com/antrea/antrea-windows
 export IMG_NAME=projects.registry.vmware.com/antrea/flow-aggregator
 ./hack/generate-manifest-flow-aggregator.sh --mode release > "$OUTPUT_DIR"/flow-aggregator.yml
 
+# Generate multicluster manifests
+export IMG_NAME=projects.registry.vmware.com/antrea/antrea-mc-controller
+multicluster/hack/generate-manifest.sh -g > "$OUTPUT_DIR"/antrea-multicluster-leader-global.yml
```

Comment:
Missing `-r` flag here?

Reply:
Hi @antoninbas  the global manifests only contains CRDs, so no need to add the flag.